app/models/concerns/can_behave_like_user.rb

```diff
@@ -47,7 +47,6 @@ module CanBehaveLikeUser
     field :locked_at,       type: Time
 
     # Validations
-    validates :password, presence: true, length: { in: 8..128 }
```

Comment:
Q: Why taking this away?

Reply:
Did you see my comment on https://github.com/DEFRA/waste-carriers-back-office/pull/569/commits/c0244cfe47869033510724b066ac4bc01b63c712?